app/src/main/java/de/westnordost/streetcomplete/data/edithistory/Edit.kt

```diff
@@ -14,7 +14,7 @@ interface Edit {
 
 sealed class EditKey
 
-data class ElementEditKey(val id: Long): EditKey()
-data class NoteEditKey(val id: Long): EditKey()
-data class OsmQuestHiddenKey(val osmQuestKey: OsmQuestKey): EditKey()
-data class OsmNoteQuestHiddenKey(val osmNoteQuestKey: OsmNoteQuestKey): EditKey()
+data class ElementEditKey(val id: Long) : EditKey()
+data class NoteEditKey(val id: Long) : EditKey()
+data class OsmQuestHiddenKey(val osmQuestKey: OsmQuestKey) : EditKey()
+data class OsmNoteQuestHiddenKey(val osmNoteQuestKey: OsmNoteQuestKey) : EditKey()
```

Comment:
This seems inconsistent to me. So here, there should be a `:`. But at 

```
try { ... } catch (e: Exception) { ... }
```
not?

And also not at 
```
fun myFun(): Boolean {
  return true
}
```
([Which rule is this? This was deliberate, to keep the ](https://github.com/streetcomplete/StreetComplete/pull/3649/files#diff-8643389ef83ba98cd5bac5c2b9cfaa0e89f78501d979a3dc97ca44ebcb737d82))

Reply:
From https://kotlinlang.org/docs/coding-conventions.html#colon

> Put a space before : in the following cases:
> - when it's used to separate a type and a supertype
> - when delegating to a superclass constructor or a different constructor of the same class
> - after the object keyword
>
> Don't put a space before : when it separates a declaration and its type.

Reply:
I have no opinion on this, just sharing where it came from.